pkg/cmd/backtest.go

```diff
@@ -597,6 +589,27 @@ var BacktestCmd = &cobra.Command{
 	},
 }
 
+func collectSubscriptionIntervals(backTestIntervals []types.Interval, requiredInterval types.Interval, environ *bbgo.Environment) (map[types.Interval]struct{}, types.Interval) {
+	allKLineIntervals := make(map[types.Interval]struct{})
+	for _, interval := range backTestIntervals {
+		allKLineIntervals[interval] = struct{}{}
+	}
+
+	for _, session := range environ.Sessions() {
+		for _, sub := range session.Subscriptions {
+			if sub.Channel == types.KLineChannel {
+				if sub.Options.Interval == types.Interval1s {
```

Comment:
I would recommend using sub.Options.Interval.Seconds() % 60 > 0 as the condition.

Reply:
what if it's 1h?